modules/admin_manual/pages/configuration/server/caching_configuration.adoc

```diff
@@ -96,9 +102,17 @@ The installer will automatically launch Redis and configure it to launch at star
 
 After that, assuming that you don’t encounter any errors, restart Apache and the extension is ready to use.
 
+==== Redis Performance Consideration
+
+* Because Redis writes data to disk, it is recommended to use a high performance backend like SSD.
+* If you do not have a distributed environment where the Redis server is installed at a different server, you can configure Redis to be xref:redis-configuration-using-unix-sockets[socket based] instead TCP based to xref:redis-performance-comparison-tcp-vs-socket[increase performance].
```

Comment:
```suggestion
* If you do not have a distributed environment where the Redis server is installed at a different machine, you can configure Redis to be xref:redis-configuration-using-unix-sockets[socket-based] instead of TCP-based to xref:redis-performance-comparison-tcp-vs-socket[increase performance].
```